tests/unit/TestPackageJson.js

```diff
@@ -12,7 +12,8 @@ const packageJson = JSON.parse(
 
 // postcss-loader: https://github.com/mozilla/addons-frontend/issues/4592
 // prettier: https://github.com/mozilla/addons-frontend/issues/5312
-const skipDevDeps = ['postcss-loader', 'prettier'];
+// pretty-quick: https://github.com/mozilla/addons-frontend/pull/5461/files#r199189927
```

Comment:
This comment doesn't need to link to a PR. Just explain it like this:

```js
// prettier and pretty-quick: we pin these so that all developers format code with the exact same version
```